### ISSUE TITLE
paddle v2怎么在预测的时候固定batch norm层的均值与方差？如何在预测时设置固定的均值和方差？

### ISSUE THREAD

**Issue by kelvict** [PaddlePaddle/Paddle#11344]
paddle v2怎么在预测的时候固定batch norm层的均值与方差？如何在预测时设置固定的均值和方差？

**Comment by Yancey1989** [comment]
可以通过`batch_norm_layer`的 `use_global_stats`参数来指定是否使用指定的均值和方差，另外预测时这个变量默认会被设置成`True`的。

另外如果想固定每个bath的均值方差不变，根据code:

https://github.com/PaddlePaddle/Paddle/blob/9328c3cf7b10c3adde66196673bcfccf908f7e05/paddle/gserver/layers/BatchNormalizationLayer.cpp#L58-L68

可以将batch_norm_layer中`moving_average_fraction`设置为1.0
